content/desktop/uninstall.md

```diff
@@ -70,6 +70,8 @@ $ rm -rf ~/Library/Containers/com.docker.docker
 $ rm -rf ~/.docker
 ```
 
+You can also move the Docker application to the trash. 
+
```

Comment:
Checked the Slack thread, it sounded like the issue was that you had to remove `/Applications/Docker.App` manually after uninstalling?

Reply:
I would just add `$ rm -rf /Applications/Docker.app` to the code block above.